euicc/es10b.h

```diff
@@ -37,7 +37,7 @@ enum es10b_error_reason
     ES10B_ERROR_REASON_INSTALL_FAILED_DUE_TO_INSUFFICIENT_MEMORY_FOR_PROFILE = 10,
     ES10B_ERROR_REASON_INSTALL_FAILED_DUE_TO_INTERRUPTION = 11,
     ES10B_ERROR_REASON_INSTALL_FAILED_DUE_TO_PE_PROCESSING_ERROR = 12,
-    ES10B_ERROR_REASON_INSTALL_FAILED_DUE_TO_ICCID_MISMATCH = 13,
+    ES10B_ERROR_REASON_INSTALL_FAILED_DUE_TO_DATA_MISMATCH = 13,
```

Comment:
```suggestion
    ES10B_ERROR_REASON_INSTALL_FAILED_DUE_TO_DATA_MISMATCH = 13,
    // For compatibility, see https://github.com/estkme-group/lpac/pull/246
    ES10B_ERROR_REASON_INSTALL_FAILED_DUE_TO_ICCID_MISMATCH = 13,
```
It only breaks API, but ABI is not broken. So we can fix it without break.
C99 explicitly allows this usage in 6.7.2.2 Enumeration specifiers.